recipes/linalg-complex/24.11/gh200/extra/reframe.yaml

```diff
@@ -0,0 +1,7 @@
+run:
+  features:
+    - dlaf
+    - cuda
+    - mpi
+  views:
+    - default
```

Comment:
I'm a bit unsure if cuda and mpi need to be listed here... Do they signal that this uenv provides cuda and mpi tests, or that the dlaf tests can run with cuda and mpi?